scripts/update-crd-reference/crd.template

```diff
@@ -1,19 +1,44 @@
 ---
-title: {{ .Title }} CRD schema reference
+title: {{ .Title }} CRD schema reference (group {{ .Group }})
```

Comment:
Adding the group name to the page title, to make CRD pages with identical short names easier to distinguish from search results.